api/controllers/order.js

```diff
@@ -23,10 +23,10 @@ router.get("/myOrders", passport.isAuthenticated(), (req, res) => {
 router.post("/createOrder", passport.isAuthenticated(), (req,res) =>{
     let content = req.body;
     let user = req.user;
-
+    console.log(content)
```

Comment:
Remember to remove this when done being used.

Reply:
I just did, thank you